tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py

```diff
@@ -36,6 +38,34 @@ def run(self):
             cluster_nodes = []
             if len(node_list) > 1:
                 # This is the master node for this flow
+                # Find number of volumes in the cluster to run profiling job
+                cmd = cmd_utils.Command('gluster volume list')
+                out, err, rc = cmd.run()
+                if not err:
+                    volumes = filter(None, out.split("\n"))
+                    ret_val, err = enable_disable_volume_profiling(
+                        volumes, self.parameters)
+                    if not ret_val:
+                        logger.log(
+                            "error",
+                            NS.publisher_id,
+                            {"message": "Failed to complete profiling "
+                                        "job. Error: %s" % err
+                             },
+                            job_id=self.parameters['job_id'],
+                            flow_id=self.parameters['flow_id']
+                        )
+                        return False
+                else:
+                    logger.log(
+                        "error",
+                        NS.publisher_id,
+                        {"message": "Failed to gather volumes info for "
```

Comment:
Failed to fetch volumes info for enable/disable profiling.

Reply:
Will do it.